modules/virt-checking-cluster-dpdk-readiness.adoc

```diff
@@ -80,6 +91,11 @@ roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: Role
   name: kubevirt-dpdk-checker
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: dpdk-checkup-traffic-gen-sa
 ----
 ====
 
```

Comment:
The following line is confusing - the manifest above contains Role, RoleBinding and ServiceAccount, but the line
` Apply the checkup roles manifest:`
implies that only roles should be applied, which gives the impression that the manifest above is not relevant.